example/config/deeprvat_input_config.yaml

```diff
@@ -0,0 +1,156 @@
+#Phenotypes to be used only for Association Testing
+phenotypes_for_association_testing:
+  - Apolipoprotein_A
+  - Apolipoprotein_B
+  - Calcium
+  - Cholesterol
+  - Red_blood_cell_erythrocyte_count
+  - HDL_cholesterol
+  - IGF_1
+  - LDL_direct
+  - Lymphocyte_percentage
+  - Mean_platelet_thrombocyte_volume
+  - Mean_corpuscular_volume
+  - Mean_reticulocyte_volume
+  - Neutrophill_count
+  - Platelet_count
+  - Platelet_crit
+  - Platelet_distribution_width
+  - SHBG
+  - Standing_height
+  - Total_bilirubin
+  - Triglycerides
+  - Urate
+
+#Phenotypes to be used only for training
+phenotypes_for_training:
+  - Apolipoprotein_A
+  - Apolipoprotein_B
+  - Calcium
+  - Cholesterol
+  - Red_blood_cell_erythrocyte_count
+
+#File paths of necessary input files to DeepRVAT
+gt_filename: genotypes.h5
+variant_filename: variants.parquet
+phenotype_filename: phenotypes.parquet
+annotation_filename: annotations.parquet
+gene_filename: protein_coding_genes.parquet
+
+rare_variant_annotations:
+  - MAF_MB
+  - CADD_raw
+  - sift_score
+  - polyphen_score
+  - Consequence_splice_acceptor_variant
+  - Consequence_splice_donor_variant
+  - Consequence_stop_gained
+  - Consequence_frameshift_variant
+  - Consequence_stop_lost
+  - Consequence_start_lost
+  - Consequence_inframe_insertion
+  - Consequence_inframe_deletion
+  - Consequence_missense_variant
+  - Consequence_protein_altering_variant
+  - Consequence_splice_region_variant
+  - condel_score
+  - DeepSEA_PC_1
+  - DeepSEA_PC_2
+  - DeepSEA_PC_3
+  - DeepSEA_PC_4
+  - DeepSEA_PC_5
+  - DeepSEA_PC_6
+  - PrimateAI_score
+  - AbSplice_DNA
+  - DeepRipe_plus_QKI_lip_hg2
+  - DeepRipe_plus_QKI_clip_k5
+  - DeepRipe_plus_KHDRBS1_clip_k5
+  - DeepRipe_plus_ELAVL1_parclip
+  - DeepRipe_plus_TARDBP_parclip
+  - DeepRipe_plus_HNRNPD_parclip
+  - DeepRipe_plus_MBNL1_parclip
+  - DeepRipe_plus_QKI_parclip
+  - SpliceAI_delta_score
+  - alphamissense
+
+covariates: #x_phenotypes
+  - age
+  - age2
+  - age_sex
+  - genetic_sex
+  - genetic_PC_1
+  - genetic_PC_2
+  - genetic_PC_3
+  - genetic_PC_4
+  - genetic_PC_5
+  - genetic_PC_6
+  - genetic_PC_7
+  - genetic_PC_8
+  - genetic_PC_9
+  - genetic_PC_10
+  - genetic_PC_11
+  - genetic_PC_12
+  - genetic_PC_13
+  - genetic_PC_14
+  - genetic_PC_15
+  - genetic_PC_16
+  - genetic_PC_17
+  - genetic_PC_18
+  - genetic_PC_19
+  - genetic_PC_20
+
+association_testing_data_thresholds:
+  MAF: "< 1e-3"
+  CADD_PHRED: "> 5"
+
+training_data_thresholds:
+  MAF: "< 1e-2"
+  CADD_PHRED: "> 5"
+
+#Seed Gene Baseline data settings
+seed_gene_results: #baseline_results
+  options:
```

Comment:
Maybe a more descriptive name for this would be better? Something like `result_dirs`?